lib/post_to_slack.rb

```diff
@@ -3,8 +3,14 @@ def self.post_slack_msg( channel, message, attachments )
     token = ENV["BOT_AUTH"]
     begin
       attachments = attachments.to_json
-      options = { query: { channel: channel, text: message, attachments: attachments }, headers: { 'Authorization' => "Bearer #{token}"} }
-      response = HTTParty.post('https://slack.com/api/chat.postMessage', options)
+      if channel==="RESULTS"
```

Comment:
add spaces `if channel === "RESULTS"`. Also, why would a string like "RESULTS" be passed to this service?

Reply:
Different API endpoints for posting to `#general` vs user/bot DMs. If it's a user/bot DM, I was sending the channel ID. To use the same function, just passed a keyword instead so it changes the URL to one set as a ENV var and option arguments accordingly.

Reply:
Oh I see. Okay, this is fine for now. In the future, we can probably refactor this a bit.

Reply:
Totally agree. I think refactors should make a separate PR though so this doesn't get crazy. 😂